doc/Development_Documentation/04_Assets/03_Working_with_Thumbnails/01_Image_Thumbnails.md

```diff
@@ -231,21 +231,6 @@ $thumbnail->getHtml([
 Additionally there are some special parameters to [customize generated image HTML code](../../03_Documents/01_Editables/14_Image.md#page_Configuration).
 
 
-## Using ICC Color Profiles for CMYK -> RGB 
```

Comment:
we should not remove that, since the feature is still there - but configuration moved to symfony config. 


```yml

pimcore:

    assets:

        # Absolute path to default ICC RGB profile (if no embedded profile is given)
        icc_rgb_profile:      null

        # Absolute path to default ICC CMYK profile (if no embedded profile is given)
        icc_cmyk_profile:     null

```